io.github.betaflight.BetaflightConfigurator.appdata.xml

```diff
@@ -35,23 +35,24 @@
   <url type="vcs-browser">https://github.com/betaflight/betaflight-configurator</url>
   <screenshots>
     <screenshot type="default">
-      <image>https://raw.githubusercontent.com/razzeee/io.github.betaflight.BetaflightConfigurator/update-deb/screenshots/Betaflight1.png</image>
+      <image>https://raw.githubusercontent.com/flathub/io.github.betaflight.BetaflightConfigurator/master/screenshots/Betaflight1.png</image>
       <caption>Welcome screen</caption>
     </screenshot>
     <screenshot>
-      <image>https://raw.githubusercontent.com/razzeee/io.github.betaflight.BetaflightConfigurator/update-deb/screenshots/Betaflight2.png</image>
+      <image>https://raw.githubusercontent.com/flathub/io.github.betaflight.BetaflightConfigurator/master/screenshots/Betaflight2.png</image>
       <caption>Drone setup screen</caption>
     </screenshot>
     <screenshot>
-      <image>https://raw.githubusercontent.com/razzeee/io.github.betaflight.BetaflightConfigurator/update-deb/screenshots/Betaflight3.png</image>
+      <image>https://raw.githubusercontent.com/flathub/io.github.betaflight.BetaflightConfigurator/master/screenshots/Betaflight3.png</image>
       <caption>Motor configuration screen</caption>
     </screenshot>
     <screenshot>
-      <image>https://raw.githubusercontent.com/razzeee/io.github.betaflight.BetaflightConfigurator/update-deb/screenshots/Betaflight4.png</image>
+      <image>https://raw.githubusercontent.com/flathub/io.github.betaflight.BetaflightConfigurator/master/screenshots/Betaflight4.png</image>
```

Comment:
don't link to master like that. Link to a specific commit.

Reply:
Good call, changed